app/components/sidebars/main/channels_list/list/list.js

```diff
@@ -414,16 +400,24 @@ export default class List extends PureComponent {
     };
 
     render() {
+        const collapsedThreadsEnabled = true;
         const {testID, styles, theme} = this.props;
         const {sections, showIndicator} = this.state;
 
         const paddingBottom = this.listContentPadding();
+        const indicatorStyle = [styles.above];
+        if (collapsedThreadsEnabled) {
+            indicatorStyle.push({marginTop: 70});
+        }
 
         return (
             <View
                 style={styles.container}
                 onLayout={this.onLayout}
             >
+                {collapsedThreadsEnabled && (
```

Comment:
seems to be used here

Reply:
Yes we are using it here but removed from the function we are passing to build the sections 🙂

Reply:
Oh sorry the diff wasn't clearly showing that it was as an arg for the selector